hw/ip/rv_dm/data/rv_dm_testplan.hjson

```diff
@@ -183,7 +183,7 @@
             - TBO - unclear how to generate other types of failed DMI transactions.
```

Comment:
Can we get rid of this line? Or is it still open, in which case we should probably create a V3(?) issue to track it?

Reply:
Good point. It isn't open. I've just pushed an extra commit that drops the line (and explains the reasoning in the commit message). Would you mind taking a quick look to check that my change is reasonable?

Reply:
Sounds perfectly reasonable to me, thx :+1: